R/REST_helpers.R

```diff
@@ -4,17 +4,16 @@
 #' @param url a base url for the POST
 #' @param body a POST body
 #' @param session a sbtools session object
-#' @param timeout a timeout for the request in seconds
-#' @param ... additional params passed to \code{\link[httr]{GET}}
+#' @param ... additional params passed to \code{\link[httr]{POST}}
 #' @export
 #' @keywords internal
-sbtools_POST <- function(url, body, session, timeout = default_timeout(), ...){
+sbtools_POST <- function(url, body, session, ...){
 	supported_types <- c('text/plain','application/json')
 	if (!session_authorized(session))
 			stop('session is not authorized. See ?authenticate_sb')
 	
 	r = POST(url, accept_json(), 
-					 body=body, handle=session, timeout(timeout)) 
+					 body=body, handle=session, config=config, ...) 
```

Comment:
i'm thinking that was a typo. my next commits won't have it. please don't merge this PR yet